src/functionalTest/java/uk/gov/hmcts/reform/pip/account/management/controllers/SystemAdminAccountTest.java

```diff
@@ -188,41 +146,27 @@ void testGetAzureUserInfo() throws Exception {
                 .contentType(MediaType.APPLICATION_JSON);
 
         MvcResult responseCreateSystemAdminUser = mockMvc.perform(createRequest)
-            .andExpect(status().isOk()).andReturn();
-
-        PiUser returnedUser = OBJECT_MAPPER.readValue(
-            responseCreateSystemAdminUser.getResponse().getContentAsString(),
-            PiUser.class
-        );
-
-        MockHttpServletRequestBuilder getRequest = MockMvcRequestBuilders
-            .get(ROOT_URL + AZURE_PATH + returnedUser.getProvenanceUserId());
-
-        MvcResult responseGetUser =
-            mockMvc.perform(getRequest).andExpect(status().isOk()).andReturn();
+            .andExpect(status().isBadRequest()).andReturn();
 
-        AzureAccount returnedAzureAccount = OBJECT_MAPPER.readValue(
-            responseGetUser.getResponse().getContentAsString(),
-            AzureAccount.class
-        );
-        assertEquals(returnedUser.getEmail(), returnedAzureAccount.getEmail(),
-                     "Should return the correct user"
+        assertEquals(BAD_REQUEST.value(), responseCreateSystemAdminUser.getResponse().getStatus(),
+                     "Should return bad request"
         );
     }
 
     @Test
-    @WithMockUser(username = "unauthroized_user", authorities = {"APPROLE_unknown.user"})
-    void testUnauthorizedCreateSystemAdminAccount() throws Exception {
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = SQL_ADD_ADMIN)
+    void testUnauthorizedUserCanCreateSystemAdminAccount() throws Exception {
```

Comment:
Can we rename thus test to testNonSystemAdminIsForbiddenToCreateSystemAdminAccount